ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_fft.py

```diff
@@ -271,6 +271,30 @@ def test_jax_numpy_ifftn(
     )
 
 
+# ifftshift
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.ifftshift",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), shape=(4,), array_api_dtypes=True
+    ),
+)
+def test_jax_numpy_ifftshift(
+    dtype_and_x, backend_fw, frontend, test_flags, fn_tree, on_device
+):
+    input_dtype, arr = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        backend_to_test=backend_fw,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=True,
+        x=arr[0],
+        axes=None,  # You can change this to test specific axes if needed
```

Comment:
This should be auto tested as in auto generated and tested for. I think there is a helper like `dtype_values_axis`
Passing a None as the arg value is testing it only for that particular case always.

Reply:
I have made the required changes , kindly have a look